tools/lib/route.py

```diff
@@ -264,7 +264,7 @@ def timestamp(self) -> str:
     return self.m.group("timestamp")
 
   @property
-  def _slice(self) -> str:
+  def slice(self) -> str:
```

Comment:
hmm, this is a builtin, but we already define similar methods in other locations without linting errors, likely because it doesn't conflict with the builtin